crates/rpc/src/api.rs

```diff
@@ -465,13 +463,15 @@ impl BeerusRpcServer for BeerusRpc {
 
     async fn get_events(
         &self,
-        filter: EventFilter,
-        continuation_token: Option<String>,
-        chunk_size: u64,
+        params: EventFilterWithPage,
```

Comment:
The spec states the parameter name as `filter`. It will work for positional parameters, but not for named ones. This also looks like a good opportunity to provide missing test case :)

(A reference to the generated code: https://github.com/eigerco/beerus/blob/5f6860cb50cd98429716667da7d2abff1e85e35f/crates/api/src/gen.rs#L3039)

Reply:
Correct. I saw that then forgot about it.

Now a problem we have is `starknet-rs` generates requests with positional parameters when the spec theoretically restricts these to named parameters only. But that's a problem for another day, and may become a non-issue if we switch to `iamgroot`.